### PR TITLE
Nralakk Passport - Borging Clarification

### PR DESCRIPTION
**Pull request by RyverStyx**
Adds wording to the Nralakk passport that reflects the Federation's views on cyborgification.

### CODE REVIEW

**Comment by RyverStyx** [comment]
!review

File: html/changelogs/DNB.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+author: RyverStyx
+
+delete-after: True
+
+changes:
+  - tweak: "Adds important info to Nralakk Federation Passports for those abroad."
```
Comment (listerla) [inline html/changelogs/DNB.yml:6]: Very ambiguously worded for some reason?

Comment (RyverStyx) [reply html/changelogs/DNB.yml]: You're right, Addressed it

**Comment by NonQueueingMatt** [comment]
Unwanted. It is expected for characters to put in the effort to write DNB on their records. Additionally, CCIA does not wish to make any one species 100% exempt from cyborgification by default.

**Pull request closed without merging**